problems/41.Implement Stack using Queues.gs

```diff
@@ -0,0 +1,110 @@
+class MyNode {
+    val: number
+    next: MyNode | null
+    constructor(value: number, nextEle: MyNode | null) {
+        this.val = value
+        this.next = nextEle
+    }
+}
+
+function add(val: number, head: MyNode | null): MyNode {
+    if (head !== null) {
+        let cur = head
+        while (cur.next !== null) {
+            cur = cur.next
+        }
+        cur.next = new MyNode(val, null)
+    } else {
+        head = new MyNode(val, null)
+    }
+    return head
+}
+function print(head: MyNode | null) {
+    let cur = head
+    while (cur != null) {
+        console.log(cur.val)
+        cur = cur.next
+    }
+}
+
+function remove(head: MyNode | null): MyNode | null {
+    if (head == null) {
+        return null
+    }
+    if (head.next == null) {
+        return null
+    }
+    if (head.next.next == null) {
+        head.next = null
+    }
+    if (head != null) {
+        let cur = head
+        if (cur.next != null && cur.next.next != null) {
+            while (cur.next?.next != null) {
+                cur = cur.next
+            }
+            cur.next = null
+        }
+        return head
+    } else {
+        return null
+    }
+}
+function onTop(head: MyNode | null): number {
+    let cur = head
+    if (cur != null) {
+        while (cur.next != null) {
+            cur = cur.next
+        }
+    }
+
+    if (cur != null) {
+        return cur.val
+    } else {
+        return -1
+    }
+
+}
+
+class MyStack {
```

Comment:
Let me help you a bit how you can clear out this mess:
1)What do you think about adding always new element by changing this.head to point to that new element instead of chaining it to the last element of the list?

Would this way be a lot more clearer and easier for all operations?
Top() is just head.value
pop() is just head = head.next
push() is just  this.head = new Elem(val, this.head);
(I didn't include checks for null for clarity sake)

All one line functions... And look what have you done